core/chains/evm/txmgr/config.go

```diff
@@ -7,18 +7,25 @@ import (
 
 	txmgrtypes "github.com/smartcontractkit/chainlink/v2/common/txmgr/types"
 	"github.com/smartcontractkit/chainlink/v2/core/assets"
-	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
+	coreconfig "github.com/smartcontractkit/chainlink/v2/core/config"
 )
 
 // Config encompasses config used by txmgr package
 // Unless otherwise specified, these should support changing at runtime
 //
 //go:generate mockery --quiet --recursive --name Config --output ./mocks/ --case=underscore --structname Config --filename config.go
 type Config interface {
-	gas.Config
+	ChainType() coreconfig.ChainType
```

Comment:
Note: I removed gas.Config from here since txmgr doesn't actually depend on gas at all. Instead an estimator is passed in at creation time. Therefore this composition doesn't actually make sense IMO.